src/resources/triton_python_backend_utils.py

```diff
@@ -85,7 +85,7 @@ def serialize_byte_tensor(input_tensor):
                     s = str(obj.item()).encode('utf-8')
             else:
                 s = obj.item()
-            flattened_ls.append(struct.pack("<I", len(s)))
+            flattened_ls.append(struct.pack("I", len(s)))
```

Comment:
super NIT: I wonder if we should be explicit here and use the `@` symbol to denote to the project in the future that this is a known decision rather than just assuming the default behavior.